public/app/features/templating/template_srv.ts

```diff
@@ -219,6 +220,27 @@ export class TemplateSrv implements BaseTemplateSrv {
     return scopedVar.value;
   }
 
+  private getVariableText(variableName: string, fieldPath: string | undefined, scopedVars: ScopedVars) {
+    const scopedVar = scopedVars[variableName];
+    if (!scopedVar) {
+      return null;
+    }
+
+    if (fieldPath) {
+      const value = this.getFieldAccessor(fieldPath)(scopedVar.value);
+
+      if (typeof value?.text === 'string') {
+        return value.text;
+      }
```

Comment:
I do not think we should support this (we do not currently), and not sure we have a use case for this